packages/lit-dev-content/site/docs/v3/components/decorators.md

```diff
@@ -85,86 +84,122 @@ In the future when decorators become a native web platform feature, this may no
 
```

Comment:
^ "In the future when decorators are supported natively in browsers, this will no longer be necessary."